cli/cmd/lql_show.go

```diff
@@ -0,0 +1,68 @@
+//
+// Author:: Salim Afiune Maya (<afiune@lacework.net>)
+// Copyright:: Copyright 2020, Lacework Inc.
+// License:: Apache License, Version 2.0
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+package cmd
+
+import (
+	"github.com/pkg/errors"
+	"github.com/spf13/cobra"
+)
+
+const (
+	lqlShowDebugMsg    string = "retrieving LQL query"
+	lqlShowNotFoundMsg string = "There were no queries found."
```

Comment:
This constant is never used. Another reason why this model is a bit confusing.

FYI, the linter will complain until you either use it or delete it. 😅
```
cli/cmd/lql_show.go:28:2: `lqlShowNotFoundMsg` is unused (deadcode)                                                                             
        lqlShowNotFoundMsg string = "There were no queries found."                                                                              
        ^                                                                                                                                       
```

Reply:
I got the linter going locally now, so we'll address this.  I thought unused things result in compiler errors.  Perhaps that's a var vs. const difference?